src/foam/nanos/script/jShell/EvalInstruction.java

```diff
@@ -73,10 +73,11 @@ private String evalSingleInstruction(String source) throws Exception {
         PrintWriter pw = new PrintWriter(sw);
         exc.printStackTrace(pw);
         output = ((EvalException) exc).getExceptionClassName() + "\n" + sw.toString();
-        System.out.println("JShell Error: " + output);
+        System.out.println("JShell Error in line : " + source );
```

Comment:
better solution, instead of grabbing the output from the EvalException and logging it, is to throw the RuntimeException with the contents of the first print as the first argument, then the exception can be rethrown. example

``
throw new RuntimeException("JShell Error in line : "+source, exc); 
``

this way, we don't need to bring in the foam logger (System.out should be avoided) and we'll also fix the ClassCastException that keeps getting thrown on line 75. kills three issues at once

Reply:
The main reason of this instruction is to present the line (instruction) that cause the issue.

Reply:
@hchoura I agree with @petenanopay regarding reporting the error.